### PR TITLE
[Validator] Updated Luxembourgish translations for 2.8

### PR DESCRIPTION
**Pull request by mweimerskirch**
| Q | A |
| --- | --- |
| Bug fix? | no |
| New feature? | no |
| BC breaks? | no |
| Deprecations? | no |
| Tests pass? | yes |
| Fixed tickets | n/a |
| License | MIT |
| Doc PR | n/a |

### CODE REVIEW

**Comment by stof** [comment]
Thank you @mweimerskirch.

File: src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf [hunk 1]
```diff
@@ -298,6 +298,22 @@
...
+                <target>En eidele Fichier ass net erlaabt.</target>
+            </trans-unit>
+            <trans-unit id="79">
+                <source>The host could not be resolved.</source>
+                <target>Den Domain-Numm konnt net opgeléist ginn.</target>
+            </trans-unit>
+            <trans-unit id="80">
+                <source>This value does not match the expected {{ charset }} charset.</source>
+                <target>Dëse Wäert entsprécht net dem erwaarten Zeechesaz {{ charset }}.</target>
+            </trans-unit>
```
Comment (WouterJ) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: 78 and 79 should be merged into 2.7 (these were added in 2.6 and 2.7) and 80 should be merged into 2.3 /cc @stof

Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: IIRC we merged all translations into `2.3` in the past to make contributing for translators easier.

Comment (fabpot) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: @xabbuh is right

Comment (WouterJ) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: The point is that they are now merged into 2.8, 3.0 and master only

Comment (xabbuh) [inline src/Symfony/Component/Validator/Resources/translations/validators.lb.xlf]: @WouterJ Sorry, I misunderstood your comment (see #16894 for the backport).

**Pull request merged**